README.md

```diff
@@ -11,29 +11,15 @@ $ pip install git+https://github.com/captain-pool/hydra-wandb-sweeper.git
 
 Using this plugin is very simple, as one can just invoke the sweeper by overriding `hydra/sweeper` with `wandb`.
 
-The only difference this plugin has with respect to other regular hydra plugin, is the function signature of `main` requires the presence of an additional positional argument called `run_id`.
-
-Hence, the final function would something like:
 ```python
 @hydra.main(config_path=..., config_name=...)
-def main(cfg: DictConfig, run_id: str): # notice the extra run_id parameter
+def main(cfg: DictConfig):
```

Comment:
There were a couple of reasons why I changed this:
1. Having a wandb.init just for retrieving a run id and config and passing it through to the task function was unnecessary overhead since you could just run the task function within the wandb.init context, as I showed in the example.
2. By removing that argument above, it keeps things in parity with what Hydra expects. Even if you were to make it an optional parameter, it avoids any weird bugs that may pop up in the future due to whatever function introspection Hydra does. Basically, I'm just trying to avoid any unnecessary problems.

In light of 1 and 2, the reason I say it's unnecessary to explicitly pass the run id is because since you're within a wandb run context already, you can just retrieve the run id by calling `wandb.run.id`.

Reply:
I don't believe running another `wandb.init()` is necessary within main since you can directly access the run by calling `wandb.run`. From there you can use any of the [setters](https://github.com/wandb/client/blob/master/wandb/sdk/wandb_run.py) within the run object. The user shouldn't be running wandb.init() again since it's just additional overhead. What they could do instead, if they want to port over their code, is to check if a currently active run is available, which would imply that they're within the run that was set from WandbTask, and from there retrieve the run and use its setters to modify things like tags, notes, etc. If they're not within a currently active run, then they can `wandb.init()`.